### DISCUSSION TITLE
How to ignore bad words in a ReStructuredText file? (rst)

### DISCUSSION THREAD

**Discussion by jugmac00** [get-woke/woke#275] [Q&A]
I use the woke linter with our documentation using sphinx and restructured text.

I have the following line, which I cannot change for semantic reasons:

```
You are now in a newly-cloned Git repository, with one branch ('master'),
```

When I use the rst comment just the line before...

```
.. 
   wokeignore:rule=master
You are now in a newly-cloned Git repository, with one branch ('master'),
```
woke is happy, but sphinx complains.

```
changes.rst:31: WARNING: Explicit markup ends without a blank line; unexpected unindent.
```

When I add the suggested empty line like this

```
.. 
   wokeignore:rule=master

You are now in a newly-cloned Git repository, with one branch ('master'),
```

Sphinx is happy, but woke complains, as the ignore rule only works for the next line, but here there is the additional empty line.

What is the recommended way to configure this properly?

For now I ignore the whole file, which works, but is not what I'd like to do.

Thank you!

**Comment by xypron** [comment]
The ignore comment must be inline.
```
You are now in a newly-cloned Git repository, with one branch ('master'), .. wokeignore:rule=master
```
